app/lib/methods/sendMessage.ts

```diff
@@ -37,33 +48,33 @@ const changeMessageStatus = async (id, tmid, status, message) => {
 	}
 
 	try {
-		await db.action(async () => {
+		await db.write(async () => {
 			await db.batch(...successBatch);
 		});
 	} catch (error) {
 		// Do nothing
 	}
 };
 
-export async function sendMessageCall(message) {
+export async function sendMessageCall(this: IRocketChat, message: TMessageModel) {
```

Comment:
It is complaining about the `this` because of the properties we're adding, not sure why since they all live on the same context; we will check on this later